### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by ninsch3000**
Adapted repo and output directory structures to match those of [PAQR2](https://github.com/zavolanlab/PAQR2).    
Miscellaneous small cleanup things.

### CODE REVIEW

File: execute/run_dag.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 snakemake \
 --use-conda \
---configfile config.yaml \
+--configfile configs/config.yaml \
```
Comment (AngryMaciek) [inline execute/run_dag.sh:5]: I don't think it matches run scripts from PAQR exactly, if I recall correctly the config was there passed as `$1`? But that's OK. As I wrote in the other PR - these things will be adjusted once I publish MAPP. This is just FYI.

**Review by dominikburri — APPROVED** [review]
Thanks for incorporating the changes :) Looks fine by me!

**Review by AngryMaciek — APPROVED** [review]
image

File: execute/run_dag.sh (hunk 1 above)
Comment (ninsch3000) [reply execute/run_dag.sh:5]: Yes, we only matched the directory structures and workflow related stuff. The run scripts are kind of an accessory...    
Great that you're gonna incorporate profiles in the future btw! :pray:

**Pull request merged**